### ISSUE TITLE
dots and underscores appended to end of the filename

### ISSUE THREAD

**Issue by ghost** [MoOx/atom-smart-tab-name#15]
![screen shot 2016-09-23 at 11 21 13](https://cloud.githubusercontent.com/assets/7355199/18781143/d63377da-8180-11e6-9fb8-f8083946db72.png)

It seems something weird is going on with dots and underscores in file- and foldernames.
- At the root of the projectfolder, dotfiles have the dot displayed to the right of their name (instead of to the left)
- Other files at the root seem to be ok
- For nested folders dotfiles do not seem to have this problem, but as you can see from the screenshot the underscores from the parent do get appended to the end

**Comment by MoOx** [comment]
Yeah this package is pretty buggy (but still better than having only "index.js" lol).
I don't have much time for this, help appreciated to improve the current render.

**Comment by MoOx** [comment]
Maybe look at #14

**Comment by ghost** [comment]
> still better than having only "index.js" lol

Definitely, thanks for creating it, much better than the atom defaults!

> I don't have much time for this, help appreciated to improve the current render.

I don't have a lot of time left at the moment either, otherwise I'd be happy to help. I'll just leave it as a bug report. I'll see if I can take a look if I have some time later.
